File: Domain/Entities/Album.cs
using System;
namespace Domain.Entities
{
    public class Album
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public AlbumType AlbumType { get; set; }
        public Guid AlbumTypeId { get; set; }
        public Artist Artist { get; set; }
        public Guid ArtistId { get; set; }
        public int Stock { get; set; }
    }
}

File: Web/src/app/models/album.ts
import { AlbumType } from './album-type';
import { Artist } from './artist';

export class Album {
  id?: string;
  artist?: Artist;
  artistId?: string;
  albumType?: AlbumType;
  albumTypeId?: string;
  title?: string;
  stock?: number;
}
